spyder/plugins/ipythonconsole/tests/test_ipythonconsole.py

```diff
@@ -92,6 +93,19 @@ def __getattr__(self, attr):
     # Tests assume inline backend
     CONF.set('ipython_console', 'pylab/backend', 0)
 
+    # Start in a new working directory the console
+    try:
+        use_startup_wdir = request.node.get_marker('use_startup_wdir')
```

Comment:
@steff456, please merge with master (after squashing some of your last commits because it seems you were tryong to fix tests with them). That will allow you to avoid using the try/except here (see PR #9818).